Dockerfile_multi_folder

```diff
@@ -0,0 +1,14 @@
+ARG BASE_IMAGE=nansencenter/geospaas:2.0.0-slim
+FROM ${BASE_IMAGE}
+RUN pip install --no-cache-dir django-celery-results==1.2
+WORKDIR /tmp/setup
+COPY setup.py README.md ./
+COPY geospaas_processing ./geospaas_processing
+RUN python setup.py bdist_wheel && \
+    pip install -v dist/geospaas_processing-*.whl && \
+    cd /tmp && rm -rf /tmp/setup/
+COPY multi_folder_download.py /multi_folder_download.py
```

Comment:
That script should rather become part of the package geospaas_processing and be installed automatically by setup.py

Reply:
I agree, and to take it a bit further, we could make a "cli" sub-package containing several modules allowing access the functionalities of geospaas_processing from a command line.

`multi_folder_download.py` can be simply renamed to `download.py`.

We can have a structure like:
```
geospaas_processing/
    cli/
        download.py
        convert.py
    ...
```

Reply:
Done in the next commit.